es/es-testing/src/main/java/org/elasticsearch/index/engine/EngineTestCase.java

```diff
@@ -622,6 +622,41 @@ protected EngineConfig config(EngineConfig config,
             tombstoneDocSupplier);
     }
 
+    protected EngineConfig config(EngineConfig config,
```

Comment:
Why another overload of `config` ? the original fix is just adjusting the existing `config?` method...

Reply:
indeed 🤦‍♂ fixed it